system/jlib/jstatcodes.h

```diff
@@ -213,6 +213,7 @@ enum StatisticKind
     StNumSmartJoinDegradedToLocal,      // number of times global smart join degraded to local smart join (<=1 unless in loop)
     StNumSmartJoinSlavesDegradedToStd,  // number of times a slave in smart join degraded from local smart join to standard hash join
     StNumAttribsSimplifiedTooComplex,
+    StNumGraphExecutions,               // number of times graph executed (<=1 unless loop or child query graph)
```

Comment:
Ideally we should have a limited number of different St kinds, and the scope would indicate that it applied to a graph.  So this should be reusing StNumExecutions (or NumIterations /NUmStarted/NumStopped). The scope indicates that it applies to a graph, so shouldn't be in the stat name.

Reply:
ok, I'll change it to StNumExecutions